.github/workflows/build.yml

```diff
@@ -76,12 +76,11 @@ jobs:
         env:
           DVC_BENCH_AZURE_CONN_STR: ${{ secrets.DVC_BENCH_AZURE_CONN_STR }}
           DATASET_SIZE: ${{ github.event.inputs.dataset }}
-        run: |
-          ./run.sh ${{ matrix.test.path }} --size ${DATASET_SIZE:-mnist}
+        run: pytest --benchmark-save ${{ matrix.test.name }} --benchmark-group-by func --dvc-revs main,2.10.0,2.9.5,2.8.3,2.6.3 ${{ matrix.test.path }} --size ${DATASET_SIZE:-mnist}
```

Comment:
It'd be convenient to specify this default on a separate file rather than in the github workflow.

Reply:
I totally agree. Just didn't bother with it for now.